tests/e2e/Services/Account/AccountBase.php

```diff
@@ -931,6 +932,58 @@ public function testUpdateAccountVerification($data): array
         return $data;
     }
 
+    /**
+     * @depends testUpdateAccountVerification
+     */
+    public function testCreateAccountVerificationForVerifiedEmail($data): array
+    {
+        $email = $data['email'] ?? '';
+        $name = $data['name'] ?? '';
+        $session = $data['session'] ?? '';
+
+        /**
+         * Test for SUCCESS
+         */
+        $response = $this->client->call(Client::METHOD_POST, '/account/verification', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
+
+        ]), [
+            'url' => 'http://localhost/verification',
+        ]);
+
+        $this->assertEquals(409, $response['headers']['status-code']);
+
+        /**
+         * Test for FAILURE
+         */
+        $response = $this->client->call(Client::METHOD_POST, '/account/verification', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
+        ]), [
+            'url' => 'localhost/verification',
+        ]);
+
+        $this->assertEquals(400, $response['headers']['status-code']);
+
+        $response = $this->client->call(Client::METHOD_POST, '/account/verification', array_merge([
+            'origin' => 'http://localhost',
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'cookie' => 'a_session_' . $this->getProject()['$id'] . '=' . $session,
+        ]), [
+            'url' => 'http://remotehost/verification',
+        ]);
+
+        $this->assertEquals(400, $response['headers']['status-code']);
```

Comment:
This is also the same as another test elsewhere, right?

Reply:
> i want error code to be 400 if email or phone no verification is called again after successful verification. Thats why two times function tests are there

But it shouldn't be 400. It should be 409 and you already have that test above 🧐 

> I saw that verification test was here so i added it

Ya, here is fine since there are already /account/verification API calls in here. And the phone verification one in the other one is good too.